src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -249,6 +251,17 @@ const validCustomText = ({ customText }) => {
   }
 };
 
+const validSaveInProgressConfig = formConfig => {
+  // TODO: Change this to not _require_ saveInProgress
+  validObjectProperty(formConfig, 'saveInProgress');
+  const messages = formConfig.saveInProgress?.messages;
+  if (messages) {
+    validStringProperty(messages, 'inProgress', false);
+    validStringProperty(messages, 'expired', false);
+    validStringProperty(messages, 'saved', false);
+  }
```

Comment:
Shouldn't we want the strings inside of `saveInProgress.messages` to be required?

Reply:
Since we have default messages, I opted for no.